Adding Flovatar Dust Token

crash13override opened the pull request:
I added both the Testnet (0x9392a4a7c3f49a0b) and the Mainnet (0x921ea449dffec68a) address, but so far, only the Testnet has been deployed.

Please let me know if I did everything right or if there is anything that I missed.

Thanks!

btspoony commented on token-registry/A.9392a4a7c3f49a0b.DustToken/token.json:
```diff
@@ -0,0 +1,17 @@
+{
```
Hi, the token list is based on the Token address of the mainnet, so there is no need for a testnet token-registry directory.
If you need to add a testnet token, you can create a `testnet.token.json` file and place it in the corresponding mainnet directory.

btspoony commented:
Also, since some production environments may directly use the token list, we need to confirm that the Token has been deployed on the mainnet before merging.

crash13override commented:
Got it, I will let you know as soon as the Token is deployed on mainnet then.
In the meantime I updated it as you suggested with the testnet.token.json file.

Does the  src/tokens/flow-mainnet.tokenlist.json file I updated do anything for testnet or is it just disregarded?

Thanks,
Luca

btspoony commented:
> 

In theory, it will be automatically generated when PR is merged... But sometimes it doesn't work...

But it doesn't matter. We can generate it by script manually...
